src/main/java/com/bookstore/bookstoreapi/data/entity/Customer.java

```diff
@@ -41,8 +41,15 @@ public class Customer {
   @Column(name = "Address", nullable = false)
   private String address;
 
+  @Column(name = "Ratings", nullable = true) //nullable = true since the user doesn't HAVE to leave ratings?
+  private ArrayList<Rating> ratings = new ArrayList<>();
+  
   @ToString.Exclude
   @Builder.Default
   @OneToMany(mappedBy = "customer", cascade = CascadeType.ALL)
   private Set<Wishlist> wishlists = new HashSet<>();
+
+  @Column(name = "Rating")
+  @OneToMany
+  private Rating rating;
```

Comment:
intead of this, you can use the `ratings` attribute to map back to the relationship by changing the:
```
  @Column(name = "Ratings", nullable = true)
  private ArrayList<Rating> ratings = new ArrayList<>();
```
to 
```
@OneToMany(mappedby = "user")
  private ArrayList<Rating> ratings = new ArrayList<>();
```
which reverses the `@ManyToOne`relationship from the rating entity.